python/requirements/lint-requirements.txt

```diff
@@ -1,4 +1,5 @@
 clang-format==12.0.1
+docutils
```

Comment:
should we pin the version?

Reply:
It's pinned in the compiled version. These non-compiled requirements should have not pin anything actually.